blob_client/src/containers/posts.js

```diff
@@ -1,6 +1,7 @@
+import React from 'react'
```

Comment:
Was this import required?

Reply:
Yes, I was getting errors without the imports.